src/main/java/redis/embedded/core/ExecutableProvider.java

```diff
@@ -35,7 +50,6 @@ static ExecutableProvider newRedis2_8_19Provider() {
         map.put(WINDOWS_x86_64, "/redis-server-2.8.19.exe");
         map.put(UNIX_x86, "/redis-server-2.8.19-32");
         map.put(UNIX_x86_64, "/redis-server-2.8.19");
-        map.put(UNIX_AARCH64, "/redis-server-2.8.19-linux-aarch64");
```

Comment:
I actually already replaced your binary with the one I generated with QEMU. It probably doesn't work but until that is confirmed I can just leave it in.

Reply:
It seems to work just fine for embedded-redis' unit tests and all unit tests of Apache Dubbo!